ROS/osr_bringup/config/roboclaw_params.yaml

```diff
@@ -1,11 +1,13 @@
 roboclaw_wrapper:
   ros__parameters:
-    drive_acceleration_factor: 0.1  # fraction used to scale the drive motor acceleration (0, 1]
+    drive_acceleration_factor: 0.8  # fraction used to scale the drive motor acceleration (0, 1]
     corner_acceleration_factor: 0.8  # fraction used to scale the corner motor acceleration (0, 1])
     velocity_timeout: 2.0  # maximum time [s] a particular velocity command will stay active without a new command before stopping
     baud_rate: 115200
     device: "/dev/serial1"
     addresses: [128, 129, 130]
+    duty_mode: true  # send duty commands to the roboclaw instead of velocity commands. This can mean a smoother drive but shouldn't be used for autonomous navigation as velocity commands aren't interpreted correclty
+    velocity_qpps_to_duty_factor: 8  # hand-tuned factor to convert velocity commands to duty commands, must be integer
```

Comment:
Request for clarification: how did you arrive at this qpps-to-duty factor?

Reply:
Like the comment says 'hand-tuned' :) 
Basically the code is written around velocity control, that's why I titled this 'naive duty mode' since I'm not really doing a rewrite to properly support duty (which isn't really torque control anyways, it's more like open loop velocity control). For example the messages passed between the rover and the roboclaw_wrapper nodes are velocities and the rover calculates velocities. Since you need a lot higher values for duty mode to get the rover to move, this magic number `velocity_qpps_to_duty_factor` is introduced. 
Only with some input from others will we be able to tell if this number needs tweaking.

Reply:
Thanks for the explanation! Makes sense to me :)